content/topics/solo-learn/sql-intermediate/3-working-with-data/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+content_type: topic
+prerequisites:
+  hard:
+  - topics/solo-learn/sql-intermediate/2-table-constraints
+  soft: []
+ready: true
+tags:
+- sql
+- solo-learn
+title: Solo Learn - SQL Intermediate - 4 Working with Data
+---
+

```

Comment:
This should be a project with a link submission since we want to verify this course's completion. See other related comment